introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<p>Citation is a little awkward - what information are you trying to display here (i.e. license? download link?)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Removed the citation.</p>

Reply:
<p>Great, this looks better - one small fix, if you need to display the license, I would put it in brackets instead of another sentence (i.e. (<a href="https://jumpstart-cache-prod-us-west-2.s3-us-west-2.amazonaws.com/licenses/Apache-License/LICENSE-2.0.txt" target="_blank">Apache 2.0 License</a>))</p>

Reply:
<p>done. Thanks.</p>